src/admin-views/aggregator/origins/ics.php

```diff
@@ -42,8 +42,8 @@ class="tribe-ea-field tribe-ea-media_button tribe-dependent button button-second
 		<button type="submit" class="button button-primary tribe-preview">
 			<?php esc_html_e( 'Preview', 'the-events-calendar' ); ?>
 		</button>
-		<button type="button" class="button tribe-clear-filters">
-			<?php esc_html_e( 'Clear Filters', 'the-events-calendar' ); ?>
+		<button type="cancel" class="button tribe-cancel">
```

Comment:
Question; Is `cancel` a type for `<button>` I've never seen it. + There isn't any JS associated with the class?